types/node/index.d.ts

```diff
@@ -3459,15 +3445,29 @@ declare module "tls" {
         minDHSize?: number;
     }
 
-    export interface Server extends net.Server {
-        close(callback?: Function): Server;
-        address(): { port: number; family: string; address: string; };
-        addContext(hostName: string, credentials: {
-            key: string;
-            cert: string;
-            ca: string;
-        }): void;
-        maxConnections: number;
+    export class Server extends net.Server {
+        /**
+         * The `server.addContext()` method adds a secure context that will be used if the client request's SNS hostname matches the supplied `hostname` (or wildcard).
+         *
+         * @param hostname A SNI hostname or wildcard (e.g. `'*'`)
+         * @param options An object containing any of the possible properties from the `tls.createSecureContext()` options arguments
+         */
+        addContext(hostName: string, options: SecureContextOptions): void;
+        /**
+         * Returns a `Buffer` instance holding the keys currently used for encryption/decryption of the TLS Session Tickets.
+         */
+        getTicketKeys(): Buffer;
+        /**
+         * Updates the keys for encryption/decryption of the TLS Session Tickets.
+         *
+         * Note: The key's Buffer should be 48 bytes long. See ticketKeys option in tls.createServer for more information on how it is used.
+         *
+         * Note: Changes to the ticket keys are effective only for future server connections. Existing or currently pending server connections will use the previous keys.
+         */
+        setTicketKeys(keys: Buffer): void;
+        /**
+         * Returns the current number of concurrent connections on the server.
+         */
         connections: number;
```

Comment:
True. It's really only here because of the docblock, but I can move that to `net`.